j2/rules/bb_mapeamento_de_processo.yaml

```diff
@@ -0,0 +1,157 @@
+          postProcessor: ${com.pontusvision.utils.BudibaseRef.fromString(it)?.getLookupId()}
+    
+    - label: Object_Data_Policy
+      props:
+        - name: Object_Data_Policy_Type
+          val: ${Referencia}
+          mandatoryInSearch: True
+        - name: Object_Data_Policy_Retention_Period
+          val: ${Periodo_de_retencao}
+          mandatoryInSearch: True
+    
+    - label: Object_Data_Procedures
+      props:
+        - name: Object_Data_Procedures_Form_Id
+          val: ${_id}
+          mandatoryInSearch: True
+        - name: Object_Data_Procedures_ID
+          val: ${Referencia}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Name
+          val: ${Processo}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Type_Of_Natural_Person
+          val: ${Categoria_dos_titulares_dos_dados}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_BusinessAreaResponsible
+          val: ${Area}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Country_Where_Stored
+          val: ${Pais_onde_as_pessoas_estao_localizadas}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Why_Is_It_Collected
+          val: ${Finalidade_do_processo}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Info_Collected
+          val: ${Dados_pessoais_utilizados}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Description
+          val: ${Fluxo_interno_dos_dados}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Data_Procedures_Interested_Parties_Consulted
+          val: ${Partes_Interessadas_Consultadas}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+    
+    - label: Object_Sensitive_Data
+      props:
+        - name: Object_Sensitive_Data_Description
+          val: ${new groovy.json.JsonBuilder(Dados_pessoais_utilizados)}
+          mandatoryInSearch: True
+          processAll: True
+          type: "[Ljava.lang.String;"
+          postProcessor: ${it?.toUpperCase()?.trim()}
+    
+    - label: Object_Lawful_Basis
+      props:
+        - name: Object_Lawful_Basis_Description
+          val: ${new groovy.json.JsonBuilder(Base_Legal)}
+          mandatoryInSearch: True
+          processAll: True
+          type: "[Ljava.lang.String;"
+          postProcessor: ${it?.toUpperCase()?.trim()}
+    
+    - label: Object_Legitimate_Interests_Assessment
+      condition: ${Is_Essential == true && Is_Required == true}
+      props:
+        - name: Object_Legitimate_Interests_Assessment_Lawful_Basis_Justification
+          val: ${N_x00e3_oh_x00e1_outrabaselegalp}
```

Comment:
this looks wrong; I can't find N_x00e3_oh_x00e1_outrabaselegalp in the budibase json.

Reply:
we'll let it pass for now, and fix it later.